drivers/vmwarevsphere/utils.go

```diff
@@ -265,3 +267,23 @@ func (d *Driver) restLogin(ctx context.Context, c *vim25.Client) (*library.Manag
 
 	return mgr, nil
 }
+
+func (d *Driver) findFolder() (*object.Folder, error) {
+	folders, err := d.datacenter.Folders(d.getCtx())
+	if err != nil {
+		return nil, err
+	}
+
+	// root vm folder default
+	folder := folders.VmFolder
```

Comment:
This can be moved down to the return statement, no reason to initialize it here since it isn't used

Reply:
sure